tests/RestServer/LanguageFeatureTest.cpp

```diff
@@ -127,6 +126,24 @@ void checkLanguageFile(const arangodb::ArangodServer& server,
     ASSERT_TRUE(false);
   }
 }
+
+constexpr std::string_view kGerman = "de";
+constexpr std::string_view kRussian = "ru";
+
+std::string_view getNonSysLang() {
+  setlocale(LC_ALL, "");
```

Comment:
This changes the locale for not only this particular test, but for all that follow. So that modifies global state.
It would be good if the test would clean up after itself and reset the locale to what it was before the test executed.